plugins/time-to-read-block/phpcs.xml.dist

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
```

Comment:
Here, I hope we could also should be able to update the shared config by setting override for the entire folder:

https://github.com/WordPress/gutenberg/blob/trunk/phpcs.xml.dist

Although, I'm not sure how to do it myself. @anton-vlasenko might be able to help.

Reply:
I looked through the [phpcs configuration docs](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset) and can't see a way to apply different rule attributes to different paths in a single command (to use different translation domains for block plugins and the main plugin in the same phpcs check under the `WordPress.WP.I18n` rule). It seems that whenever you redeclare the same ruleset, it overrides any previous instances of the same rule. I believe we could add extra domains to the list, but that doesn't seem like enough, as it wouldn't catch `'gutenberg'` being used in block plugins.

So the best I could come up with so far
- Move most of the rules to a `phpcs.xml.shared` file
- Import the shared rules into the main `phpcs.xml.dist`, and into a separate file for each block plugin
- Use a script when running the `lint:php` command that does a separate run of phpcs for each `phpcs.xml.dist` config

But let me know if I'm missing something or there's a better way to organize this.